catalog/core/catalog-core-downloadaction/src/main/java/org/codice/ddf/catalog/download/action/ResourceDownloadActionProvider.java

```diff
@@ -130,7 +130,7 @@ private boolean hasRemoteSiteName(Metacard metacard) {
   }
 
   String getLocalSiteName() {
-    return SystemInfo.getSiteName().toLowerCase();
+    return SystemInfo.getSiteName();
```

Comment:
❓ If we are using `equalsIgnoreCase` everywhere, should we really change the data? There might be other reasons we are converting to lowercase.